fixit/rules/chained_instance_check.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, Set
+
+import libcst as cst
+import libcst.matchers as m
+
+from fixit import (
+    CstContext,
+    CstLintRule,
+    InvalidTestCase as Invalid,
+    ValidTestCase as Valid,
+)
+
+
+class ChainedInstanceCheckRule(CstLintRule):
```

Comment:
We hope the lint rule name is short and actionable. So developer can learn the action to take easily.
https://fixit.readthedocs.io/en/latest/build_a_lint_rule.html#Pick-a-Good-Name

What do you think about rename as`CollapseIsinstanceChecksRule`?
```suggestion
class CollapseIsinstanceChecksRule(CstLintRule):
```

Reply:
`Isinstance` doesn't look so nice (personally would prefer `IsInstance`, but not a big deal), but OK, sure!